### PR TITLE
#6 - cve_fixes_updated_dependencies

### LINKED ISSUE #6: Dependency check build fails
```
Error:  Failed to execute goal org.sonatype.ossindex.maven:ossindex-maven-plugin:3.2.0:audit (default-cli) on project performance-test-recorder-java: Detected 1 vulnerable components:
Error:    org.apache.commons:commons-text:jar:1.9:compile; https://ossindex.sonatype.org/component/pkg:maven/org.apache.commons/commons-text@1.9?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
Error:      * [CVE-2022-42889] CWE-94: Improper Control of Generation of Code ('Code Injection') (9.8); https://ossindex.sonatype.org/vulnerability/CVE-2022-42889?component-type=maven&component-name=org.apache.commons%2Fcommons-text&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
```

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #6

### CODE REVIEW

File: doc/changes/changes_0.1.2.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Performance Test Recorder Java 0.1.2, released 2022-10-26
+
+Code name: Dependency check build fails : updated dependencies
+
+## Summary
+
+Dependency check build failed : updated dependencies to fix issue.
```
Comment (kaklakariada) [inline doc/changes/changes_0.1.2.md:7]: Please list the fixed vulnerabilities

File: doc/changes/changes_0.1.2.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Performance Test Recorder Java 0.1.2, released 2022-10-26
+
+Code name: Dependency check build fails : updated dependencies
```
Comment (kaklakariada) [inline doc/changes/changes_0.1.2.md:3]:
```suggestion
Code name: Updated dependencies
```

File: doc/changes/changes_0.1.2.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Summary
+
+Dependency check build failed : updated dependencies to fix issue.
+
+## Features
+
+* #6:Dependency check build fails : updated dependencies
```
Comment (kaklakariada) [inline doc/changes/changes_0.1.2.md:11]:
```suggestion
* #6: Dependency check build fails: updated dependencies
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.1.2.md (hunk 1 above)
Comment (pj-spoelders) [reply doc/changes/changes_0.1.2.md:7]: Done

**Pull request merged**